### PR TITLE
Updates README.md with typo fix

### PR DESCRIPTION
**Pull request by crispy8888**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,7 +44,6 @@ If you are working on the download functionality and are having problems without
 
 The two `xcconfig` files are used to configure the project. To access the values specified, these files must be added to the __Info.plist__ file.
 
-Use the `Configuration` struct to access these values from code.
```
Comment (sammyd) [inline README.md]: Why has this line gone?

**Review by sammyd — COMMENTED** [review]
Not sure why the line was deleted

File: README.md (hunk 1 above)
Comment (crispy8888) [reply README.md]: Oh! It shouldn't have.

**Review by sammyd — APPROVED** [review]
:+1: merge when you desire

**Pull request merged**